inference-engine/src/vpu/graph_transformer/src/stages/roi_align.cpp

```diff
@@ -75,6 +78,9 @@ class ROIAlignStage final : public StageNode {
         }
 
         outputEdge(0)->output()->serializeBuffer(serializer);
+        const auto use_buffer = attrs().get<bool>(s_use_buffer);
```

Comment:
We can use `tempBuffers.empty()` or `numTempBuffers() > 0` instead of attribute. What do you think about it?

Reply:
Yes, it is more simple, used it. Thank you!